overops-functions/src/main/java/com/takipi/udf/microsoftteams/card/MicrosoftPotentialAction.java

```diff
@@ -0,0 +1,56 @@
+package com.takipi.udf.microsoftteams.card;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MicrosoftPotentialAction {
+    @SerializedName("@type")
+    public String type;
+    public String name;
+    public List<MicrosoftTarget> targets;
+
+    public MicrosoftPotentialAction(String type, String name, List<MicrosoftTarget> targets) {
+        this.type = type;
+        this.name = name;
+        this.targets = targets;
+    }
+
+    public static Builder newBuilder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        private String type = "OpenUri";
+        private String name = "";
+        private List<MicrosoftTarget> targets = new ArrayList<>();
+
+        public Builder setType(String type) {
+            this.type = type;
+            return this;
+        }
+
+        public Builder setName(String name) {
+            this.name = name;
+            return this;
+        }
+
+        public Builder setTargets(List<MicrosoftTarget> targets) {
+            this.targets = targets;
+            return this;
+        }
+
+        public Builder addTargets(MicrosoftTarget... targets) {
+            if (this.targets == null) this.targets = new ArrayList<>();
```

Comment:
Please have `{}` surrounding `if` code blocks, even if it's just a single line.
In this case:
```
if (this.targets == null) {
    this.targets = new ArrayList<>();
}
```

Reply:
done